File: app/src/main/java/com/ddpc/ggway/ui/activity/home/MainHomeView.java
package com.ddpc.ggway.ui.activity.home;

import com.ddpc.ggway.ui.MainView;

/**
 * Created by diha- on 10.02.2018.
 */

public interface MainHomeView extends MainView {
    void changeFragment(int position);
    void startSignInActivity();
    void showAlertDialogUpdateProfileInformation();
}

File: app/src/main/java/com/ddpc/ggway/ui/activity/home/MainHomeView.kt
package com.ddpc.ggway.ui.activity.home

import com.ddpc.ggway.ui.MainView

/**
 * Created by diha- on 10.02.2018.
 */

interface MainHomeView : MainView {
    fun changeFragment(position: Int)
    fun startSignInActivity()
    fun showAlertDialogUpdateProfileInformation()
}
